sink/src/main/java/com/spredfast/kafka/connect/s3/sink/S3SinkTask.java

```diff
@@ -69,6 +74,17 @@ public void start(Map<String, String> props) throws ConnectException {
         .map(Long::parseLong)
         .ifPresent(chunkThreshold -> this.GZIPChunkThreshold = chunkThreshold);
 
+    configGet("compressed_file_size")
+        .map(Long::parseLong)
+        .ifPresent(gzipThreshold -> this.GZIPFileThreshold = gzipThreshold);
+
+    configGet("flush.interval.ms")
+        .map(Long::parseLong)
+        .ifPresent(flushIntervalMs -> this.flushIntervalMs = flushIntervalMs);
+
+    log.info("{} compressed_file_size: {}", name(), GZIPFileThreshold);
+    log.info("{} flush.interval.ms: {}", name(), flushIntervalMs);
```

Comment:
These log messages are derived from the configuration. No need to produce them in production.